sbin/submitty_autograding_worker.py

```diff
@@ -27,6 +27,8 @@
 
 JOB_ID = '~WORK~'
 
+ALL_WORKERS_JSON = os.path.join(SUBMITTY_DATA_DIR, "autograding_TODO", "autograding_worker.json")
```

Comment:
what is the point of this file when we've got `${SUBMITTY_INSTALL_DIR}/config/autograding_worker.json`?